yarn.lock

```diff
@@ -4275,6 +4290,11 @@ clone@^1.0.2:
   resolved "https://registry.yarnpkg.com/clone/-/clone-1.0.4.tgz#da309cc263df15994c688ca902179ca3c7cd7c7e"
   integrity sha1-2jCcwmPfFZlMaIypAheco8fNfH4=
 
+clsx@^1.0.1:
```

Comment:
That's interesting. Is this maybe responsible for the increased bundle size?
Edit: Nope: `matererial-ui-pickers` uses this.